lib/workers/repository/config-migration/index.ts

```diff
@@ -1,29 +1,36 @@
 import type { RenovateConfig } from '../../../config/types';
 import { logger } from '../../../logger';
+import type { Pr } from '../../../modules/platform';
 import { checkConfigMigrationBranch } from './branch';
 import { MigratedDataFactory } from './branch/migrated-data';
 import { ensureConfigMigrationPr } from './pr';
 
+export interface ConfigMigrationResult {
+  result?: 'add-checkbox' | 'pr-exists';
+  prNumber?: number;
+}
```

Comment:
I think prNumber doesn't need to be returned from the `checkConfigMigrationBranch` function, because we again fetch the pr info using the `ensureConfigMigrationPr` below.